lib/iris/tests/integration/experimental/geovista/test_cube_to_poly.py

```diff
@@ -0,0 +1,66 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the BSD license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Integration tests for the `iris.experimental.geovista.cube_to_polydata` function."""
+
+import numpy as np
+
+from iris import load_cube
+from iris.experimental.geovista import cube_to_polydata
+from iris.experimental.ugrid import PARSE_UGRID_ON_LOAD
+from iris.tests import get_data_path
+
+
+def test_integration_2d():
```

Comment:
I've done some rough plotting to demonstrate this is doing the right thing:

### GeoVista

![Screenshot 2024-03-27 145003](https://github.com/SciTools/iris/assets/40734014/2a915d7f-2b34-4b08-a0de-d14ce1e828aa)




### Matplotlib (for comparison)

![Screenshot 2024-03-27 150126](https://github.com/SciTools/iris/assets/40734014/70875fb4-5ca9-4289-b89b-55d916bac2ba)